blockchain/genesis/genesis.go

```diff
@@ -76,6 +76,7 @@ func defaultConfig() Genesis {
 			RedseaBlockHeight:       26704441,
 			SumatraBlockHeight:      28516681,
 			TsunamiBlockHeight:      29275561,
+			UpernavikBlockHeight:    39275561,
```

Comment:
the hardfork height will be reset later, right?

Reply:
right, we set to a very far date, before release, will change to the planned date